### ISSUE TITLE
Error when running proxy command

### ISSUE THREAD

**Issue by JamieIsGeek** [LOOHP/InteractiveChat#196]
Hi, whenever I run a command for a plugin running on my proxy, InteractiveChat seems to be intercepting and stopping the command producing this error:

```
[12:28:53 ERROR]: Exception in thread "Timer-0" java.lang.IllegalArgumentException: Can not set com.velocitypowered.proxy.protocol.packet.chat.ComponentHolder field com.velocitypowered.proxy.protocol.packet.chat.SystemChatPacket.component to net.kyori.adventure.text.TextComponentImpl
[12:28:53 ERROR]:       at java.base/jdk.internal.reflect.FieldAccessorImpl.throwSetIllegalArgumentException(FieldAccessorImpl.java:228)
[12:28:53 ERROR]:       at java.base/jdk.internal.reflect.FieldAccessorImpl.throwSetIllegalArgumentException(FieldAccessorImpl.java:232)
[12:28:53 ERROR]:       at java.base/jdk.internal.reflect.MethodHandleObjectFieldAccessorImpl.set(MethodHandleObjectFieldAccessorImpl.java:115)
[12:28:53 ERROR]:       at java.base/java.lang.reflect.Field.set(Field.java:836)
[12:28:53 ERROR]:       at com.loohp.interactivechat.proxy.velocity.InteractiveChatVelocity.lambda$onProxyInitialization$4(InteractiveChatVelocity.java:304)
[12:28:53 ERROR]:       at com.loohp.interactivechat.proxy.objectholders.ProxyMessageForwardingHandler$1.run(ProxyMessageForwardingHandler.java:120)
[12:28:53 ERROR]:       at java.base/java.util.TimerThread.mainLoop(Timer.java:566)
[12:28:53 ERROR]:       at java.base/java.util.TimerThread.run(Timer.java:516)
```

Running on velocity and the backend paper servers :)

**Comment by LOOHP** [comment]
Is this still happening in IC 4.2.10?

**Comment by T14D3** [comment]
No, fixed with 4.2.10
